Floofbot/Modules/Fun.cs

```diff
@@ -52,10 +49,33 @@ public async Task AskEightBall([Summary("question")][Remainder] string question)
             };
             Random random = new Random();
             int randomNumber = random.Next(responses.Count);
+            return responses[randomNumber];
+        }
+
+        public static string RespondEightBall(string question)
+        {
+            Regex questionRe = new Regex(@"^.*\?$");
+            if(questionRe.IsMatch(question)) {
+                string response = Floofbot.Modules.Fun.EightBallResponse();
+                if (!string.IsNullOrEmpty(response))
+                {
+                    string reply = $"> {question}";
+                    reply += "\n";
+                    reply += $"{response}";
+                    return reply;
+                }
+            }
+            return "";
+        }
+
+        [Command("8ball")]
+        [Summary("Ask the Magic 8-Ball a question")]
+        public async Task AskEightBall([Summary("question")][Remainder] string question)
+        {
             EmbedBuilder builder = new EmbedBuilder();
             builder.Title = "Magic 8 Ball";
             builder.AddField("Question", question);
-            builder.AddField("Answer", responses[randomNumber]);
+            builder.AddField("Answer", EightBallResponse());
```

Comment:
`new EightBall().Response` would work better here once the class has been made.
Look into C# properties since those are used instead of Java-style getters and setters https://www.dotnetperls.com/property

Reply:
Created a static helper class